src/EntityFramework/Core/Metadata/Edm/Provider/ClrProviderManifest.cs

```diff
@@ -79,6 +79,9 @@ internal static bool TryGetPrimitiveTypeKind(Type clrType, out PrimitiveTypeKind
                     case TypeCode.Byte:
                         primitiveTypeKind = PrimitiveTypeKind.Byte;
                         break;
+                    case TypeCode.Char:
```

Comment:
adding Char to the provider manifest might have significant impact in many places. Please remove this and just keep the overload that takes string escapeCharacter.